src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -81,16 +83,33 @@
             <ui:outputText value="{!$Label.c.stgHelpContactAddrs}" class="slds-text-body--small" />
         </div>
 
+        <div class="slds-col slds-size--1-of-2">
+            <ui:outputText value="{!$Label.c.stgAdminAccountRecordType}" class="slds-text-body--small" />
+        </div>
+        <div class="slds-col slds-size--1-of-2">
+            <c:CMP_RecTypes_Dropdown class="hh-addresses-account-record-type"
+                                     setting="{!v.hierarchySettings.Administrative_Account_Record_Type__c}"
+                                     isView="{!v.isView}"
+                                     recTypeId="{!v.adminAccRecTypeId}"
+                                     recTypeName="{!v.adminAccRecTypeName}"
+                                     recTypes="{!v.accRecTypes}"
+            />
+        </div>
+
+        <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
+            <ui:outputText value="{!$Label.c.stgHelpAdminRecType}" class="slds-text-body--small" />
+        </div>
+
         <div class="slds-col slds-size--1-of-2">
             <ui:outputText value="{!$Label.c.stgAccountRecordTypeSupportsHHAddress}" class="slds-text-body--small" />
         </div>
         <div class="slds-col slds-size--1-of-2">
             <c:CMP_RecTypes_Dropdown class="hh-addresses-account-record-type"
-              setting="{!v.hierarchySettings.Household_Addresses_RecType__c}"
-              isView="{!v.isView}"
-              recTypeId="{!v.householdRecTypeId}"
-              recTypeName="{!v.householdRecTypeName}"
-              recTypes="{!v.accRecTypes}"
+                                     setting="{!v.hierarchySettings.Household_Addresses_RecType__c}"
```

Comment:
just checking for the spacing here. looks like it may have been indented really far.

Reply:
I was trying to align those attributes but I can change it back if needed. :)